eng/pipelines/common/platform-matrix.yml

```diff
@@ -229,6 +229,7 @@ jobs:
       helixQueuesTemplate: ${{ parameters.helixQueuesTemplate }}
       osGroup: Windows_NT
       archType: arm
+      platform: Windows_NT_arm
```

Comment:
This led us down the path of not actually adding a windows_nt_arm queue. Which is why our ci currently skips windows arm32

Reply:
Oops. Maybe we should error out if Queues are empty? Rather than just skipping that step? 

That's what happens for Libraries and it catches errors like that.